timescale-cloud/maintenance.md

```diff
@@ -0,0 +1,51 @@
+# Maintenance
+On Timescale Cloud, software updates are handled automatically by us, and you do not need to perform any actions.
+
+Non-critical software updates are applied during a maintenance window that you can define to suit your workload. If we detect a security vulnerability that affects you, we might need to perform maintenance outside of the scheduled maintenance window.
+
+<highlight type="important">
+During a maintenance update, the DNS name remains the same, but the IP address
```

Comment:
I would change "During" to "After". "During" suggests that the IP address is different only during the maintenance window, but I assume that the restart that happens as a result of maintenance means it now has a different IP address going forward (until the next maintenance).